### DISCUSSION TITLE
멘티 - 멘토 요청 성립 이후 가이드가 필요합니다.

### DISCUSSION THREAD

**Discussion by JuneParkCode** [manito42/FrontEnd#31] [General]
현재 프론트 진행상황 공유가 되지 않은 상태라, 과정이 조금 모호해서 질문드립니다.
멘토가 멘토링 요청 수락 시 어떤 페이지가 보여지는걸까요?

멘토의 경우 멘토링 요청 수락 이후 저희 서비스에서 멘티에게 연락을 유도해야합니다. (예를 들어, "멘티에게 슬랙 메시지를 보내보세요~" 이런거)
멘티의 경우에도 멘토링 요청 수락을 알림 받고, 멘토에게 연락을 할 수 있도록 해야합니다. (일단은 intra profile 링크를 주고 슬랙을 보내도록 유도하려고 합니다.)

이런 가이드가 없으면 "어떻게 연락하는거야" 라고 좀 혼선이 생길 것 같다는 생각이 듭니다.
혹시 해당 부분이 처리되었다면 샘플을 댓글로 남겨주시면 감사하겠습니다.

**Comment by JuneParkCode** [comment]
@nyeoni @9utty

**Comment by 9utty** [comment]
가이드는 따로 있지않습니다. 해당 유저의 닉네임과 같이 멘토링 신청이 왔습니다 정도에서 끝난다고 보시면 될 것 같습니다. 승인도 똑같습니다! (ex: ooo님과 멘토링 진행중입니다.) 이런식으로 reservation(enroll) list에서 보여줍니다.
